awx/main/tests/functional/task_management/__init__.py

```diff
@@ -0,0 +1,6 @@
+def create_job(jt, dependencies_processed=True):
+    job = jt.create_unified_job()
+    job.status = "pending"
```

Comment:
I think the canonical way to do this would be `signal_start` with some mocks on top. I'm not trying to change anything now, just want to fill in any gaps in understanding.